R/appenders.R

```diff
@@ -255,6 +255,34 @@ appender_syslog <- function(identifier, ...) {
 }
 
 
+#' Send log messages to a network syslog server
+#' @param identifier program/function identification (string).
+#' @param server machine where syslog daemon runs (string).
+#' @param port port where syslog daemon listens (integer).
+#'
+#' @return A function taking a \code{lines} argument.
+#' @export
+#' @note This functionality depends on the \pkg{syslognet} package.
+#' @examples \dontrun{
+#' if (requireNamespace("syslognet", quietly = TRUE)) {
+#'   log_appender(appender_syslognet("test_app", 'remoteserver'))
+#'   log_info("Test message.")
+#' }
+#' }
+appender_syslognet <- function(identifier, server, port = 601L) {
+  fail_on_missing_package('syslognet')
```

Comment:
we still need to force the `appender_syslognet` params to be passed down to the closure:

```suggestion
  fail_on_missing_package('syslognet')
  force(server)
  force(port)
```